eZ/Publish/Core/Repository/Helper/NameSchemaService.php

```diff
@@ -237,7 +237,8 @@ protected function getFieldTitles(array $schemaIdentifiers, $contentType, array
                     foreach ($contentType->fieldDefinitions as $spiFieldDefinition) {
                         if ($spiFieldDefinition->identifier === $fieldDefinitionIdentifier) {
                             $fieldDefinition = $this->contentTypeDomainMapper->buildFieldDefinitionDomainObject(
-                                $spiFieldDefinition
+                                $spiFieldDefinition,
+                                $languageCode
```

Comment:
whoops, I meant to ask about that.
No, this is not a main language. Main language at this point, for `SPIContentType` is available only via `initialLanguageId`. 

So, to get actual mainLanguageCode here it would require injecting Persistence Language Handler into NameSchemaService and retrieve it via `$languageHandler->load($contentType->initialLanguageId)->languageCode` which, I guess, wouldn't be efficient.

Currently we use created here `fieldDefinition` to get field settings only, so code won't fail. We could pass `null` there, but IMHO it's even worse than passing some existing language code.

I'm not sure what's the best approach here.